### DISCUSSION TITLE
How to remove gravity from acceleration or how to measure velocity or distance from accelation

### DISCUSSION THREAD

**Discussion by Butinam** [jrowberg/i2cdevlib#611] [Q&A]
I am doing a vr project and I need to calculate the position using the mpu sensor.

So, I have to calculate the distance from the acceleration.

I use the output quaternion values to rotate well but I can't move object.

So, I want the value to be 0, 0, 0 remove gravity when the sensor is not moving.

I am using arduino(mpu 9250) and Unity.

Please give me some advice.

**Comment by abdelrahma22** [comment]
hi Butinam if u solve this problem please share with me

**Comment by bingcheng1998** [comment]
:+1:
